woocommerce/woocommerce-ios
# [REST API] Migrate shipment tracking provider endpoints

Closes: #8742


## Description
This PR migrates the shipment tracking provider endpoints and updates the related mappers to parse contents without the data envelope.


## Testing instructions
Prerequisite
- You need to have https://woocommerce.com/products/shipment-tracking/ plugin installed on your site.

Steps
- Enable the feature flag `applicationPasswordAuthenticationForSiteCredentialLogin` and build the app.
- Log out of the app or skip onboarding if needed.
- On the prologue screen, select Enter your site address and proceed with the address of a self-hosted store.
- Log in with your site credentials. After the login succeeds, you should be navigated to the home screen.
- Navigate to Orders tab and open an Order
- Scroll down and tap on "Add Tracking"
- Select a shipping carrier and add a random tracking number and tap "Add"
- You should be able to add tracking as before.
- Once added, try deleting the tracking by tapping on "..." -> "Delete Tracking"

## Screenshots

![Simulator Screen Recording - iPhone 14 Pro - 2023-01-25 at 09 51 09](https://user-images.githubusercontent.com/524475/214489105-5505cd34-5a7d-4a45-9ecd-7070d6a7f0a4.gif)


---
